frontend/src/bundles/common/components/loader/loader.tsx

```diff
@@ -0,0 +1,34 @@
+import { type Color, type Size } from '~/bundles/common/types/types.js';
+
+import { LogoIcon } from '../icons/icons.js';
+
+type Properties = {
+    isOverflow?: boolean;
+    color?: Color;
+    size?: Size;
+};
+
+const Loader: React.FC<Properties> = ({
+    isOverflow,
+    color = 'primary',
+    size = 'lg',
+}: Properties): JSX.Element => {
+    const spinner = (
+        <>
+            <span aria-hidden="true">
+                <LogoIcon color={color} size={size} className="animate-load" />
+            </span>
+            <span className="sr-only">Loading...</span>
+        </>
+    );
+
+    return isOverflow ? (
+        <div className="absolute flex h-full w-full items-center justify-center">
+            {spinner}
+        </div>
+    ) : (
+        spinner
+    );
+};
```

Comment:
I think we can simplify it like this
```
<div className={getValidClassNames(isOverflow && "absolute flex h-full w-full items-center justify-center")}>
            {spinner}
 </div>
```
You can merge the button branch in order to get this helper

Reply:
I've noticed the bug that since I use div in every situation (both in isOverflow and !isOverflow) I get the block behavior which is not desirable. I've already changed the `classNames` definition